lawnchair/res/values-de-rDE/strings.xml

```diff
@@ -201,6 +201,8 @@
     <string name="color_dark">Dunkel</string>
     <string name="force_rounded_widgets">Abgerundete Ecken</string>
     <string name="allow_widget_overlap">Überlappen zulassen</string>
+    <string name="force_widget_resize">Alle Widgets skalieren</string>
+    <string name="widget_unlimited_size">Beliebige Größe für Widgets</string>
```

Comment:
👋 Head up, Crowdin CI will remove your changes on Sunday after this PR merged.

Consider translating the strings once this PR is merged at https://lawnchair.crowdin.com/lawnchair

Reply:
Good to know, thank you! I intend to correct a mistake in the German translation anyway.